lib/twitter.js

```diff
@@ -161,7 +161,7 @@ Twitter.prototype.post = function(url, content, content_type, callback) {
     }
     else {
       try {
-        var json = JSON.parse(data || {});
+        var json = data ? JSON.parse(data) : data;
```

Comment:
I was thinking `var json = data ? JSON.parse(data) : {} ;` but as I say down there, I'm ok with this too, just thinking out loud :+1: